.gitignore

```diff
@@ -1,3 +1,6 @@
 /venv/
 /.idea
-/__pycache__/
\ No newline at end of file
+/__pycache__/
+.venv
+tool_output.txt
```

Comment:
Keep .gitignore to generic files and IDE dir/files

Reply:
![image](https://github.com/Suterahana/suterakira/assets/45171340/28f73d2d-251b-47be-b306-b6dce84b58f0)